src/__test__/utils/pathwayAnalysis/launchPathwayService.test.js

```diff
@@ -4,18 +4,35 @@ import { pathwayServices } from 'utils/pathwayAnalysis/pathwayConstants';
 
 const genesList = {
   gene_names: ['gene1', 'gene2'],
+  gene_id: ['ENMUSG00000001', 'ENMUSG00000002'],
 };
-const species = 'sapiens';
+const enrichrSpecies = 'sapiens';
+const pantherDBSpecies = 'HUMAN';
 
 describe('LaunchPathwayService test', () => {
+  beforeAll(() => {
+    // This is required because JSDOM doesn't implement form.submit()
+    // https://github.com/jsdom/jsdom/issues/1937#issuecomment-461810980
+    window.HTMLFormElement.prototype.submit = () => {};
+  });
+
+  it('Launches the pantherDB service', () => {
+    // If the launch goes well, it should return undefined
+    expect(
+      launchPathwayService(pathwayServices.ENRICHR, genesList, enrichrSpecies),
+    ).toBe(undefined);
+  });
+
   it('Launches the enricher service', () => {
-    // If it all goes well, it should return undefined
-    expect(launchPathwayService(pathwayServices.ENRICHR, genesList, species)).toBe(undefined);
+    // If the lalunch goes well, it should return undefined
+    expect(
+      launchPathwayService(pathwayServices.PANTHERDB, genesList, pantherDBSpecies),
+    ).toBe(undefined);
```

Comment:
Because the function doesn't return anything on success, but will throw an error if it fails. Therefore, if the launch goes well, it should return undefined.

Reply:
[here](https://github.com/hms-dbmi-cellenics/ui/pull/610#discussion_r786876791)